src/main/java/org/broadinstitute/consent/http/service/DarCollectionService.java

```diff
@@ -647,7 +647,7 @@ public DarCollection cancelDarCollectionElectionsAsChair(DarCollection collectio
     // Cancel filtered DAR elections
     cancelElectionsForReferenceIds(referenceIds);
 
-    return darCollectionDAO.findDARCollectionByCollectionId(collection.getDarCollectionId());
+    return getByCollectionId(collection.getDarCollectionId());
```

Comment:
This ensures that datasets are populated so we can log dar cancellations for the chair case